client/app/lib/helpers/url-helpers.js

```diff
@@ -21,5 +21,25 @@ function getUrlParameter(sParam) {
   return '';
 }
 
+/**
+ * Get the course id from URL.
+ *
+ * return {number}
+ */
+function getCourseId() {
+  const match = window.location.pathname.match(/^\/courses\/(\d+)/);
+  return match && match[1];
+}
+
+/**
+ * Get the survey id from URL.
+ *
+ * return {number}
+ */
+function getSurveyId() {
```

Comment:
Do you mean to do something like `getId('survey')` (as part of the exported API) or just have an internal function that varies the regex? If it is the former, will we lose the ability to verify that the id getter for a given item (e.g. course, survey) exist at transpile/bundle time? Not sure if that is a concern, but we can always write tests for it instead.